workers.aqua

```diff
@@ -6,7 +6,7 @@ import PeerId from "./builtin.aqua"
 service Worker("worker"):
     -- Creates new worker associated with `deal_id`.
     -- Throws an error if worker exists.
-    create(deal_id: string) -> PeerId
+    create(deal_id: string, cu_ids: []string) -> PeerId
```

Comment:
please add doc explaining what is `cu_ids`

Reply:
done